src/main/java/org/springframework/integration/kafka/inbound/KafkaMessageSource.java

```diff
@@ -90,8 +92,6 @@
  */
 public class KafkaMessageSource<K, V> extends AbstractMessageSource<Object> implements Pausable {
 
-	private static final long DEFAULT_POLL_TIMEOUT = 50L;
```

Comment:
N.B. The default poll timeout value from `ConsumerProperties` is now 5000 ms. This is significantly higher than what it was previously. Consequently, the assign timeout value is now going to be 100 s.

Do we need to preserve the existing values?

Reply:
Good point; I didn't think about that - I think we need a boolean on `ConsumerProperties` to indicate whether it was explicitly set, or defaulted. In the latter case we can override it to 50ms.

Reply:
And what is the problem to be consistent with listener container and rely on whatever is default in the `ConsumerProperties`?